From 61e6b2b88a58278a4433fb8c3b40b6040229178d Mon Sep 17 00:00:00 2001
From: Mike Pape <dotzenlabs@gmail.com>
Date: Thu, 31 Jan 2008 09:12:12 -0500
Subject: [PATCH] Export all neon symbols for now.

Otherwise, we get an error because of a double | | from libtool.

Signed-off-by: Mike Pape <dotzenlabs@gmail.com>
---
 neon/configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/neon/configure.in b/neon/configure.in
index 15bebf1..af11a40 100644
--- a/neon/configure.in
+++ b/neon/configure.in
@@ -150,7 +150,7 @@ fi
 
 # Library-internal symbols are in the ne__ namespace: tell libtool
 # to not export these from the built library if possible.
-NEON_LINK_FLAGS="$NEON_LINK_FLAGS -export-symbols-regex '^ne_[[^_]]'"
+NEON_LINK_FLAGS="$NEON_LINK_FLAGS"
 
 if test x${enable_shared}${pic_mode}z = xnodefaultz; then
    CFLAGS="$CFLAGS -prefer-pic"
-- 
1.6.3.2.1299.gee46c

